tests/integration/models/application.spec.coffee

```diff
@@ -13,6 +13,79 @@ describe 'Application Model', ->
 
 	givenLoggedInUser()
 
+	describe 'rolling release related methods', ->
+
+		beforeEach ->
+			resin.models.application.create
+				name: 'FooBar'
+				applicationType: 'microservices-starter'
+				deviceType: 'raspberry-pi'
+			.then (application) =>
+				@application = application
+				userId = application.user.__id
+
+				resin.pine.post
+					resource: 'release'
+					body:
+						belongs_to__application: @application.id
+						is_created_by__user: userId
+						commit: 'old-release-commit'
+						status: 'success'
+						source: 'cloud'
+						composition: {}
+						start_timestamp: 1234
+
+				.then =>
+					resin.pine.post
+						resource: 'release'
+						body:
+							belongs_to__application: @application.id
+							is_created_by__user: userId
+							commit: 'new-release-commit'
+							status: 'success'
+							source: 'cloud'
+							composition: {}
+							start_timestamp: 54321
+
+
+		it 'should disable and enable rolling release', ->
+
+			resin.models.application.disableRollingRelease(@application.id)
+			.then =>
+				promise = resin.models.application.isRollingReleaseEnabled(@application.id)
+				m.chai.expect(promise).to.eventually.be.false
+
+			.then =>
+
+				resin.models.application.enableRollingRelease(@application.id)
+			.then =>
+				promise = resin.models.application.isRollingReleaseEnabled(@application.id)
+				m.chai.expect(promise).to.eventually.be.true
+
+		describe 'setApplicationToRelease()', ->
+
+			it 'should throw the error if rolling release is enabled', ->
+
+				resin.models.application.enableRollingRelease(@application.id)
+				.then =>
+					promise = resin.models.application.isRollingReleaseEnabled(@application.id)
+					m.chai.expect(promise).to.eventually.be.true
```

Comment:
We don't need to retest this here I think. We've got a separate test above that already checks that `enableRollingRelease` works, so we can skip this step imo.